src/test/groovy/graphql/nadel/engine/ResultNodesToOverallResultTest.groovy

```diff
@@ -0,0 +1,8 @@
+package graphql.nadel.engine
+
+import spock.lang.Specification
+
+class ResultNodesToOverallResultTest extends Specification {
+
```

Comment:
Seems bare

Reply:
I don' wanna test it for now because I wanna wait until we integrated hydration into it to see better the overall design.